tests/fixtures/scopes.js

```diff
@@ -0,0 +1,18 @@
+export const scopes = [
```

Comment:
scopes -> scopesList, to understand, that this is array of values

Reply:
done